proto/sequencerapis/astria/sequencer/v1alpha1/service.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package astria.sequencer.v1alpha1;
+
+import "astria/sequencer/v1alpha1/block.proto";
+
+message GetSequencerBlockRequest {
+  // The height of the block to retrieve.
+  int64 height = 1;
+}
+
+message GetFilteredSequencerBlockRequest {
```

Comment:
See my suggestion above to add an `optional bytes rollup_id` field to `GetSequencerBlockRequet`.

Other than that I think I liked your original naming suggestion of calling this `GetRollupSpecificSequencerBlockRequest`. Although wordy it's much more concrete than `Filtered` which sounds very general.